site/content/in-dev/unreleased/entities.md

```diff
@@ -53,6 +53,10 @@ Polaris tables are entities that map to [Apache Iceberg tables](https://iceberg.
 
 For information on managing tables with the REST API or for more information on what data can be associated with a table, see [the API docs]({{% github-polaris "client/python/docs/CreateTableRequest.md" %}}).
 
+## Generic Table
```

Comment:
Generic tables aren't their own entity type. Instead, can you amend the section above to clarify that tables may map to either Iceberg tables or generic tables?